src/components/models-download/AddModelDownloadModal.tsx

```diff
@@ -82,7 +79,7 @@ export default function AddModelDownloadModal() {
                     <span className="card-title mb-0 label-text flex"><span className="cursor-help underline filtersUnderline">Name</span></span>
                 </Tooltip>
                 <Dropdown2 options={filteredList && filteredList} useDifferentTextColor={colorDownloadedModels} differentTextColor="green" valuePropertyPath="configString"
-                    hasSearchBar={true} dropdownItemsClasses="max-h-96 overflow-y-auto" ignoreDisabledForSearch={true}
```

Comment:
why was ignoreDisabledForSearch remoed?
I added it intentioanlly

Reply:
The ignoreDisabledForSearch was causing a build error. Seems like an non-existing prop.

Reply:
have you merged the submodules? it's part of the react-components (new propery to the Dropdown2)

Reply:
Oops! Oversight on my side! I will re-add them now and test it again.